packages/expo-firebase-analytics/package.json

```diff
@@ -32,6 +32,9 @@
   "author": "650 Industries, Inc.",
   "license": "MIT",
   "homepage": "https://github.com/expo/expo/tree/master/packages/expo-firebase-analytics",
+  "jest": {
+    "preset": "expo-module-scripts/ios"
```

Comment:
This appears deprecated. Was there a reason to use the `/ios` preset specifically?

```suggestion
    "preset": "expo-module-scripts"
```


As per `expo-module-scripts/ios/jest-preset`:

```js
const createJestPreset = require('../createJestPreset');

console.warn(
  'The Jest preset "expo-module-scripts/ios" is deprecated, please convert your tests to universal tests and use "expo-module-scripts" instead'
);
module.exports = createJestPreset(require('jest-expo/ios/jest-preset'));
```

Reply:
Let me remove `iOS`. Honestly just a copy from another package. 

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FoCCLHVNt8YO64%2Fgiphy.gif&token=copy" data-gitmeme-token="copy"><img src="https://media2.giphy.com/media/oCCLHVNt8YO64/giphy.gif" title="Created by gitme.me with /copy"/></a>